packages/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -263,47 +263,54 @@ const useRootFocusStyles = makeStyles({
   // used by the outline pseudo-element. Need to introduce a text container for children and set
   // overflow there so that default focus outline can work
   //
-  // base: theme => createFocusOutlineStyle(theme),
+  // base: createFocusOutlineStyle(),
   // circular: theme =>
-  //  createFocusOutlineStyle(theme, { style: { outlineRadius: theme.global.borderRadius.circular } }),
-  // primary: theme => createFocusOutlineStyle(theme, { style: { outlineOffset: '2px' } }),
-  // square: theme => createFocusOutlineStyle(theme, { style: { outlineRadius: theme.global.borderRadius.none } }),
+  //  createFocusOutlineStyle({ style: { outlineRadius: theme.global.borderRadius.circular } }),
+  // primary: theme => createFocusOutlineStyle({ style: { outlineOffset: '2px' } }),
+  // square: theme => createFocusOutlineStyle({ style: { outlineRadius: theme.global.borderRadius.none } }),
 
-  base: createCustomFocusIndicatorStyle(theme => ({
-    ...shorthands.borderColor('transparent'),
-    outlineColor: 'transparent',
-    outlineWidth: '2px',
-    outlineStyle: 'solid',
-    boxShadow: `
+  base: theme =>
+    createCustomFocusIndicatorStyle({
+      ...shorthands.borderColor('transparent'),
+      outlineColor: 'transparent',
+      outlineWidth: '2px',
+      outlineStyle: 'solid',
+      boxShadow: `
       ${theme.shadow4},
       0 0 0 2px ${theme.colorStrokeFocus2}
     `,
-    zIndex: 1,
-  })),
+      zIndex: 1,
+    }),
 
-  circular: createCustomFocusIndicatorStyle(theme => ({
-    ...shorthands.borderRadius(theme.borderRadiusCircular),
-  })),
+  circular: theme =>
+    createCustomFocusIndicatorStyle({
+      ...shorthands.borderRadius(theme.borderRadiusCircular),
+    }),
```

Comment:
Presumably you're going to replace `theme` with `tokens` here in a future PR?

Reply:
Yes, this is only changing the usages inside tabster or inside of the tabster functions (i.e. `createCustomFocusIndicatorStyle(theme => ...)`. I'm changing everything else in separate PRs that take a look at each specific package more deeply.